test1/task1/main.c

```diff
@@ -0,0 +1,84 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stddef.h>
+
+
+int isPalindrome(char str[], int len)
+{
+    int i = 0;
+    int j = len - 1;
+    while (i < j)
+    {
+        if (str[i] != str[j])
+        {
+            return 0;
+        }
+        ++i;
+        --j;
+    }
+    return 1;
+}
+
+int test()
+{
+    char palindroms[2][6] = {"123321", "sossos"};
+    char notPalindroms[2][6] = {"123456", "sososo"};
+    if (!isPalindrome(palindroms[0], 6) || !isPalindrome(palindroms[1], 6))
+    {
+        return 0;
+    }
+    if (isPalindrome(notPalindroms[0], 6) || isPalindrome(notPalindroms[0], 6))
```

Comment:
```suggestion
    if (isPalindrome(notPalindroms[0], 6) || isPalindrome(notPalindroms[1], 6))
```
Видимо, так предполагалось.